contracts/CBKLand.sol

```diff
@@ -38,7 +41,7 @@ contract CBKLand is Initializable, ERC721Upgradeable, AccessControlUpgradeable {
     }
 
     function _restricted() internal view {
-        require(hasRole(GAME_ADMIN, msg.sender), "Not game admin");
+        require(hasRole(GAME_ADMIN, msg.sender) || hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "NA");
```

Comment:
is this only for the convenience of console admins? it's a nitpick over a principle, probably only like 100 gas so I'm not too bothered, just wondering